.vscode/settings.json

```diff
@@ -1,10 +1,21 @@
 {
+    "editor.codeActionsOnSave": {
+        "source.fixAll.eslint": true
+    },
     "editor.defaultFormatter": "esbenp.prettier-vscode",
     "editor.formatOnSave": true,
     "eslint.options": {
-        "extensions": [".md", ".mdx"]
+        "extensions": [".md", ".mdx"],
```

Comment:
I found that the plugin threw an error when it encountered this `extension` option, to remove this